src/renderer/components/header/Header.tsx

```diff
@@ -21,8 +23,14 @@ export const Header: React.FC = (): JSX.Element => {
   } = midgardService
   const midgardUrl = useObservableState(apiEndpoint$, RD.initial)
 
-  const { explorerUrl$ } = useBinanceContext()
-  const explorerUrl = useObservableState(explorerUrl$, O.none)
+  const binanceUrl$ = useBinanceContext().explorerUrl$
+  const binanceUrl = useObservableState(binanceUrl$, O.none)
+
+  const bitcoinUrl$ = useBitcoinContext().explorerUrl$
+  const bitcoinUrl = useObservableState(bitcoinUrl$, O.none)
+
+  const thorchainUrl$ = useThorchainContext().explorerUrl$
+  const thorchainUrl = useObservableState(thorchainUrl$, O.none)
```

Comment:
[hint] Use `getExplorerAddressByChain` (see its usage in (Settings)[https://github.com/thorchain/asgardex-electron/blob/4413dc931b995d957f4b26efc78ef76f4dbba2aa/src/renderer/views/wallet/SettingsView.tsx#L197-L202]. The reason for this is to get rid of `useXYZClientContext` in the future ...

Reply:
^ `getExplorerAddressByChain` is not `explorerUrl` - my bad - just ignore it.